### PR TITLE
Updated test case  - OCP27609

### PR DESCRIPTION
**Pull request by miyadav**
…https://bugzilla.redhat.com/show_bug.cgi?id=1831717

*Removing the step of copying yaml to modify

V3 runner - https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/93069/console

### CODE REVIEW

**Comment by miyadav** [comment]
@jhou1  @sunzhaohua2  PTAL

File: features/machine/machine.feature [hunk 1]
```diff
@@ -128,21 +128,12 @@ Feature: Machine features testing
...
-    When I run the :replace admin command with:
-     | f | new_machineset.yaml |
-    And the output should match:
-     | [Rr]eplaced |
+    Then I run the :patch admin command with:
+     | resource      | machineset                                         |
+     | resource_name | machineset-clone-27609                             |
+     | type          | merge                                              |
+     | p             | {"metadata":{"providerSpec": {"publicIP": "true"}}}|
+    And the step should succeed
```
Comment (liangxia) [inline features/machine/machine.feature:136]: Given /^(?:(as admin) )?I successfully#{OPT_WORD} patch resource "(.*)\/(.*)" with:$/ do |as_admin, patch_type, resource_type, resource_name, table|

Comment (miyadav) [reply features/machine/machine.feature]: Thanks @liangxia  for review , made changes as suggested and validated them , looking good - https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/93838/console 

Please help merge this

File: features/machine/machine.feature [hunk 2]
```diff
@@ -125,26 +125,11 @@ Feature: Machine features testing
...
-    Then the step should succeed
-
-    When I run the :replace admin command with:
-     | f | new_machineset.yaml |
-    And the output should match:
-     | [Rr]eplaced |
-
-    Given I scale the machineset to +2
+    Then as admin I successfully merge patch resource "machineset/machineset-clone-27609" with:
+     | {"spec":{"template": {"spec":{"providerSpec":{"value":{"publicIP": "true"}}}}}} |
```
Comment (liangxia) [inline features/machine/machine.feature:131]: Indention issue. Generally we use two white spaces.

File: features/machine/machine.feature [hunk 3]
```diff
@@ -125,26 +125,11 @@ Feature: Machine features testing
     And I switch to cluster admin pseudo user
     And I use the "openshift-machine-api" project
     And admin ensures machine number is restored after scenario
-
+
```
Comment (liangxia) [inline features/machine/machine.feature:128]: Perhaps we do not need the change on this line ?

File: features/machine/machine.feature (hunk 2 above)
Comment (miyadav) [reply features/machine/machine.feature]: done

File: features/machine/machine.feature (hunk 3 above)
Comment (miyadav) [reply features/machine/machine.feature]: yes , removed that space

**Pull request merged**